app/models/manageiq/providers/automation_manager.rb

```diff
@@ -11,7 +11,12 @@ class ManageIQ::Providers::AutomationManager < ManageIQ::Providers::BaseManager
 
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
-  has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :configuration_scripts,
+           :dependent   => :destroy,
+           :foreign_key => "manager_id",
+           :class_name  => "::ConfigurationScript",
```

Comment:
this looks redundant, the class should be inferred automatically

Reply:
The inferred class would be `AutomationManager::ConfigurationScript`.  The requirement here is to return both `AutomationManager::ConfigurationScript` and `AutomationManager::ConfigurationWorkflow`